test/healthcheck.test.js

```diff
@@ -18,17 +18,22 @@ describe('healthcheck', function() {
           assert(res.body.started);
           assert(res.body.uptime);
           assert.equal(res.body.version, 'x.x.1');
+          assert.strictEqual(res.body.env, undefined);
           done();
         });
     });
   });
   it('should install route passed in params when registered via API', function(done) {
     const app2 = loopback();
     app2.middleware(
-      'routes',
+      'routes:before',
       healthcheck({
         path: '/test',
-        versionFile: './package.json'
+        versionFile: './package.json',
+        env: {
```

Comment:
why not just pass a list instead, 
```suggestion
        env: ['TEST_VAR_1', 'TEST_VAR_2]
```

Reply:
It seems it's using a object to let you define the alias of the VAR name used for component. e.g
```
env: { 'alias1': 'VAR_NAME_1' }
```

while in deployment the env var is named as VAR_NAME_1.